doc/source/serve/deployment-graph.md

```diff
@@ -16,15 +16,15 @@ kernelspec:
 
 # Deployment Graph
 
-```{note} 
```

Comment:
trailing whitespace, hard to revert but good for code quality, so i kept these changes.